src/main/resources/configCivcraft.yml

```diff
@@ -9487,7 +9472,7 @@ recipes:
     input:
       emeralds:
         material: EMERALD
-        amount: 5
+        amount: 10
```

Comment:
Eh ok. I don't care a ton it just seems 100% anathema with our discussion

Keeping lower tier shit cheap should stay; I'd rather recommend making the other stuff that's higher cost cheaper then in the lower tiers.